packages/cli/.vscode/launch.json

```diff
@@ -59,6 +59,24 @@
         "${workspaceFolder}/../api/build/**/*.js"
       ],
       "console": "integratedTerminal"
+    },
+    {
+      "type": "pwa-node",
+      "request": "launch",
+      "name": "Launch permission status command (non-interactive)",
+      "skipFiles": ["<node_internals>/**"],
+      "program": "${workspaceFolder}/cli.js",
+      "args": ["permission", "status", "--folder", "../../../teams/app001"],
```

Comment:
What's the teams/app001 folder?

Reply:
When running cli, should specify the target folder. This folder follows the same pattern as above.

Reply:
OK, when local debug, do we really need this folder?